Libraries/LibWeb/CSS/Parser/ValueParsing.cpp

```diff
@@ -2538,6 +2539,34 @@ Optional<ShapeRadius> Parser::parse_shape_radius(TokenStream<ComponentValue>& to
     return {};
 }
 
+RefPtr<FitContentStyleValue> Parser::parse_fit_content_value(TokenStream<ComponentValue>& tokens)
+{
+    auto transaction = tokens.begin_transaction();
+    auto& component_value = tokens.consume_a_token();
+
+    if (component_value.is_ident("fit-content"sv)) {
+        transaction.commit();
+        return FitContentStyleValue::create();
+        return nullptr;
+    }
+
+    if (!component_value.is_function())
+        return nullptr;
+
+    auto const& function = component_value.function();
+    if (function.name != "fit-content"sv)
+        return nullptr;
+    if (function.value.size() != 1)
```

Comment:
This doesn't account for whitespace inside the function. Instead you want to `argument_tokens.discard_whitespace()`, parse_length_percentage(), discard whitespace again, and reject it if there are any argument_tokens left.